R/calcVoPcrops.R

```diff
@@ -15,7 +16,7 @@
 #' a <- calcOutput("VoPcrops")
 #' }
 #'
-calcVoPcrops <- function(fillGaps = TRUE) {
+calcVoPcrops <- function(fillGaps = TRUE, unit="constant 2005 US$MER") {
   # Value of production of individual items (US$MER05)
   item <- "Gross_Production_Value_(USDMER05)_(1000_US$)"
```

Comment:
convertFAO_online already got the 2017 conversion so needs to be adjusted here... 
the item name will also be "Gross_Production_Value_(USDMER17)_(1000_US$)"
see here: https://github.com/caviddhen/mrfaocore/blob/gdp_update/R/convertFAO_online.R#L347